roles/edpm_fips/tasks/disable.yml

```diff
@@ -0,0 +1,64 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Disable FIPS
```

Comment:
i dont think we need disable mode like this

disabled can be achieved by just not including the OpenstackDataplaneService that runs this

alternatively disable could be implemented as just skipping with a debug message stating "fips management is disabled".

we do not intend to support converting a fips enabled host to a non fips enabled host.

Reply:
There could be a future legitimate requirement to switch fips off during deploy, such as when we only distribute one image which is fips enabled, but a 3rd party extension doesn't support it.

Reply:
I don't see any reason not to include disable functionality, partly for the reason @steveb notes.

Reply:
My doubts are strictly related to the command itself. Let me quote the man page:

```
       •   --disable: Undo some of the FIPS-enablement steps. Please note that
           module installation cannot be undone without reformatting of and
           overwriting, at least once, the platform’s hard drive or other
           permanent storage media. This option is not meant to be used in
           production, is not supported, and is implemented for testing
           purposes only.
```

Should we trust it? Will it be enough?

Reply:
As above, reading the code, it just add `fips=0` to the bootloader options. Should we do something else?